docs/cli-usage.md

```diff
@@ -0,0 +1,17 @@
+# CLI Usage
```

Comment:
@pabloest Note that I don't include this in the navigation tree

Reply:
I am fine with excluding it but how do I get to this doc in general?

Reply:
It linked from the Semgrep README and appears in search results